app/Http/Controllers/FrontController.php

```diff
@@ -168,7 +168,7 @@ protected function getRelatedSidebarItemsData()
 
         $view['html'] = view('site.shared._featuredRelated', [
             'item' => $item,
-            'autoRelated' => $this->getAutoRelated($item),
+            'autoRelated' => ($item->toggle_autorelated ?? true) ? [] : $this->getAutoRelated($item),
```

Comment:
I'm not following the logic here. I would expect `$item->toggle_autorelated ?? true` to always evaluate to `true`:

* if `$item->toggle_autorelated` is true then the block is true
* if `$item->toggle_autorelated` is false then `?? true` will make the block fall back to true. 

In which case `? []` will always be returned, meaning this will always get an empty array. I'm not sure `$this->getAutoRelated($item)` will ever get called. 

Testing The Bedroom on my local box, however, the two autorelated highlights are displayed when the Supress checkbox is left unchecked. But how?

Reply:
@nikhiltri since $item->toggle_autorelated is a boolean it will return true or false and then continue. I think I initially had this code setup to be default to true in case the value wasn't set to avoid failures but then I went through and added the casts & attributes to each model so by default autorelated is not toggled